Makefile

```diff
@@ -457,42 +453,6 @@ fossa-test:
 
 build-fossa: bins fossa-analyze fossa-delay fossa-test
 
-##### Docker #####
-docker-server:
```

Comment:
I don't see this block in the new repo. It is convenient to build images locally.

Reply:
added https://github.com/temporalio/docker-builds/pull/16